build/browser/app.js

```diff
@@ -9148,7 +9154,7 @@ angular.module("template/typeahead/typeahead-popup.html", []).run(["$templateCac
 !angular.$$csp() && angular.element(document).find('head').prepend('<style type="text/css">.ng-animate.item:not(.left):not(.right){-webkit-transition:0s ease-in-out left;transition:0s ease-in-out left}</style>');
 },{}],8:[function(require,module,exports){
 /**
- * @license AngularJS v1.4.8
+ * @license AngularJS v1.5.0-beta.2
```

Comment:
Can you omit the angular version bump change from this commit, to keep it clean? I'll send a new PR later on upgrading the dependencies.